PL2/PL2-shared/src/main/java/nl/tudelft/pl2016gr2/model/OriginalGraph.java

```diff
@@ -15,8 +15,17 @@ public OriginalGraph() {
   }
 
   /**
-   * Print a string representation of this graph.
+   * Construct an OriginalGraph.
```

Comment:
There should be a blank line in between the description and the tags. Not sure why checkstyle doesn't pick this up, but there are only limited checkstyle rules for JavaDoc